WatchParty/backend/src/server.js

```diff
@@ -1,11 +1,29 @@
+require('dotenv').config();
 const express = require('express');
 const app = express();
 const port = process.env.PORT || 5000;
-
+const {Client}=require('pg');
+const {Sequelize} = require('sequelize');
+var config = require('./configuration/config/config.js');
 // console.log that your server is up and running
 app.listen(port, () => console.log(`Listening on port ${port}`));
 
 // create a GET route
 app.get('/express_backend', (req, res) => {
     res.send({ express: 'YOUR EXPRESS BACKEND IS CONNECTED TO REACT' });
 });
+
+const sequelize = new Sequelize(config.test.database,config.test.username,config.test.password,{
```

Comment:
Change this to `config.database`, `config.username`, etc.. and make `config.js` take these from the env variables instead. Would make it easier to set them on the servers, and for dev we could have them set in a .env before running.

Reply:
each variable in config does take from the env variables. It goes from env -> config -> server.js

Reply:
I mean right now it's enforcing that we can only use the test database. I think it would be better to do something like
`const sequelize = new Sequelize(config.database,config.username,config.password`
and let the env decide what those are. So for development on local machine, all .env files will have
`config.database` equal to the old `config.test.database`
whereas I can set the env variables on production such that
`config.database` uses the old `config.deploy.database`